src/main/resources/application.yml

```diff
@@ -0,0 +1,18 @@
+server:
+  port: 8080
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    password: postgres
+    username: postgres
+    url: jdbc:postgresql://db-wish-list/wish-list
+  jpa:
```

Comment:
No need to mave to yml file. Properties are still good

Reply:
Yaml is pretty and support features of collections, multi-line strings, etc

Reply:
> Yaml is pretty and support features of collections, multi-line strings, etc

Overkill